driver-adapters/neon-http-node-basic/index.js

```diff
@@ -22,134 +23,137 @@ exports.handler = async () => {
-      }
-    }),
-    updateMany: await prisma.user.updateMany({
-      where: {
-        age: 26
-      },
-      data: {
-        age: 27
       },
     }),
+    // updateMany: await prisma.user.updateMany({
+    //   where: {
+    //     age: 26,
+    //   },
+    //   data: {
+    //     age: 27,
+    //   },
+    // }),
     findFirst: await prisma.user.findFirst({
       where: {
-        age: 27
+        age: 27,
       },
       select: {
         email: true,
         age: true,
         name: true,
-      }
-    }),
-    delete: await prisma.user.delete({
-      where: {
-        email: "test-1@prisma.io"
       },
-      select: {
-        email: true,
-        age: true,
-        name: true,
-      }
     }),
+    // delete: await prisma.user.delete({
+    //   where: {
+    //     email: 'test-1@prisma.io',
+    //   },
+    //   select: {
+    //     email: true,
+    //     age: true,
+    //     name: true,
+    //   },
+    // }),
     count: await prisma.user.count(),
-    aggregate: await prisma.user.aggregate({
-      _avg: {
-        age: true
-      },
-    }).then(({ _avg }) => ({ age: Math.trunc(_avg?.age ?? 0) })),
+    aggregate: await prisma.user
+      .aggregate({
+        _avg: {
+          age: true,
+        },
+      })
+      .then(({ _avg }) => ({ age: Math.trunc(_avg?.age ?? 0) })),
     groupBy: await prisma.user.groupBy({
       by: ['age'],
       _count: {
-        age: true
+        age: true,
       },
     }),
     findFirstOrThrow: await prisma.user.findFirstOrThrow({
       select: {
         age: true,
         email: true,
         name: true,
-      }
+      },
     }),
     findUniqueOrThrow: await prisma.user.findUniqueOrThrow({
       where: {
-        email: "test-2@prisma.io"
+        email: 'test-1@prisma.io',
       },
       select: {
         age: true,
         email: true,
         name: true,
-      }
-    }),
-    upsert: await prisma.user.upsert({
-      where: {
-        email: "test-4@prisma.io"
-      },
-      create: {
-        email: "test-4@prisma.io",
-        age: 30,
-        name: 'Test 4',
       },
-      update: {},
-      select: {
-        email: true,
-        age: true,
-        name: true,
-      }
     }),
+    // upsert: await prisma.user.upsert({
+    //   where: {
+    //     email: 'test-4@prisma.io',
+    //   },
+    //   create: {
+    //     email: 'test-4@prisma.io',
+    //     age: 30,
+    //     name: 'Test 4',
+    //   },
+    //   update: {},
+    //   select: {
+    //     email: true,
+    //     age: true,
+    //     name: true,
+    //   },
+    // }),
   })
 
   const regResult = await getResult(prisma)
-  
+
   // Transactions are expected to fail in HTTP mode
   try {
     const itxResult = await prisma.$transaction(getResult)
   } catch (e) {
-    if (e.message !== 'Transactions are not supported in HTTP mode') {
+    if (e.code !== 'P2036' && e.message !== 'PrismaClientKnownRequestError: Error in external connector (id 0)') {
```

Comment:
Why no match on the error message from before? Huh?

Reply:
Me as well, because that is what was output!?

Reply:
See added test here https://github.com/prisma/ecosystem-tests/pull/4054/files#r1350495275